### PR TITLE
#483 adjust reporting for isLess/GreaterThanOrEqual

### PR DESCRIPTION
**Pull request by Russiancold**
`Replace with` works fine. Hope that formatting is not too messy

______________________________________
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/master/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

**Comment by Russiancold** [comment]
Any suggestions?)

**Comment by robstoll** [comment]
Thanks already, I'll review tomorrow and let you know if there are things which need to be done in addition.

File: translations/de_CH/atrium-translations-de_CH-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionComparableAssertion.kt [hunk 1]
```diff
@@ -8,7 +8,22 @@ import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
...
 enum class DescriptionComparableAssertion(override val value: String) : StringBasedTranslatable {
     IS_LESS_THAN("ist weniger als"),
+
+    @Deprecated("Use IS_LESS_THAN_OR_EQUALS instead; Will be removed in 1.0.0", ReplaceWith("DescriptionComparableAssertion.IS_LESS_THAN_OR_EQUALS"))
     IS_LESS_OR_EQUALS("ist weniger oder gleich"),
+
+    /**
+     * @since 0.12.0
+     */
+    IS_LESS_THAN_OR_EQUALS("ist weniger als oder gleich wie"),
```
Comment (robstoll) [inline translations/de_CH/atrium-translations-de_CH-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionComparableAssertion.kt:18]:
```suggestion
    IS_LESS_THAN_OR_EQUAL("ist weniger als oder gleich wie"),
```

File: translations/de_CH/atrium-translations-de_CH-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionComparableAssertion.kt [hunk 2]
```diff
@@ -8,7 +8,22 @@ import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
...
+
     IS_GREATER_THAN("ist grösser als"),
+
+    @Deprecated("Use IS_GREATER_THAN_OR_EQUALS instead; Will be removed in 1.0.0", ReplaceWith("DescriptionComparableAssertion.IS_GREATER_THAN_OR_EQUALS"))
     IS_GREATER_OR_EQUALS("ist grösser oder gleich"),
+
+    /**
+     * @since 0.12.0
+     */
+    IS_GREATER_THAN_OR_EQUALS("ist grösser als oder gleich wie"),
```
Comment (robstoll) [inline translations/de_CH/atrium-translations-de_CH-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionComparableAssertion.kt:28]:
```suggestion
    IS_GREATER_THAN_OR_EQUAL("ist grösser als oder gleich wie"),
```

**Comment by robstoll** [comment]
@Russiancold do you need help?

**Comment by robstoll** [comment]
@Russiancold I close the PR as it seems you are not working on it any more. Feel free to re-open and rebase on master (or simply start over).

**Pull request closed without merging**